server/src/services/series.js

```diff
@@ -37,6 +39,86 @@ const aggregateSeries = async (seriesIds) => {
     ]);
 };
 
+const filterSeries = async ({ name, status, genre }, pageNumber, pageLimit) => {
+    const aggregationQuery = [];
+
+    name && aggregationQuery.push({ $addFields: { searchIndex: { $indexOfCP: ["$name", String(name)] } } }, { $match: { searchIndex: { $ne: -1 } } });
```

Comment:
When I searched name="W" I found: "Wednesday", "The Witcher: Blood Origin", "Willow", "The Walking Dead", "The Witcher", "S.W.A.T."
I believe this filter should work as startsWith logic rather than substring, as originally implemented, if I think from the user's perspective.

Reply:
I actually think that this kind of search is better than startWith.. from user's perspective - not everybody remember how the series name begin, it's more likely to remember part of the name